src/main/java/TextView.java

```diff
@@ -20,9 +20,13 @@
 	public void update(Observable o, Object arg)
```

Comment:
- General Board Display
    - [ ] The board spacing seems a little off. Each column is a little to the right of the column label.
    - [ ] The top and bottom boundaries of the grid do not extend fully to the right.
- General Notes
    - [ ] Instances that show up throughout:
        - [ ] `System.out.println("");` I'm guessing you are using this to print a new line? Instead define `final static String NEWLINE = "\n";` as an attribute of the class. Then replace the first bit with `System.out.println(NEWLINE);`, or use `NEWLINE` as part of a StringBuilder.
        - [ ] `System.out.println("  ");` Similar to above. Instead define `final static String SPACE = "  ";` as an attribute of the class. Then replace the first bit with `System.out.println(SPACE);`